apps/web/components/ui/avatar/UserAvatarGroupWithOrg.tsx

```diff
@@ -1,35 +1,32 @@
-import { useOrgBranding } from "@calcom/features/ee/organizations/context/provider";
-import { CAL_URL, WEBAPP_URL } from "@calcom/lib/constants";
+import { WEBAPP_URL } from "@calcom/lib/constants";
 import { getUserAvatarUrl } from "@calcom/lib/getAvatarUrl";
+import { getBookerBaseUrlSync } from "@calcom/lib/getBookerUrl/client";
 import type { Team, User } from "@calcom/prisma/client";
 import { AvatarGroup } from "@calcom/ui";
 
 type UserAvatarProps = Omit<React.ComponentProps<typeof AvatarGroup>, "items"> & {
-  users: Pick<User, "organizationId" | "name" | "username">[];
+  users: (Pick<User, "organizationId" | "name" | "username"> & { bookerUrl: string })[];
   organization: Pick<Team, "slug" | "name">;
 };
 
 export function UserAvatarGroupWithOrg(props: UserAvatarProps) {
   const { users, organization, ...rest } = props;
-  const orgBranding = useOrgBranding();
```

Comment:
`useOrgBranding` gives the org details for the logged in user and not the organizer being booked.